eregs_core/static/regulations/js/source/models/meta-model.js

```diff
@@ -120,7 +120,12 @@ var MetaModel = Backbone.Model.extend({
         }
 
         if (typeof this.supplementalPath !== 'undefined') {
-            url += this.supplementalPath + '/';
+            if (this.supplementalPath == 'sidebar') {
+                url = '/sidebar_partial/'
+            }
+            else {
+                url += this.supplementalPath + '/';
```

Comment:
On further reflection it seems like it's just easier to change the backend routes because of the way that the frontend is structured. So I'll update this PR to do that instead.